packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss

```diff
@@ -60,7 +139,12 @@ $chat-typingindicator-circle-bg-color-center: null !default;
   $chat-messagelist-empty-icon-background-color: lighten($base-bg, 7.84) !default;
   $chat-messagelist-empty-prompt-color: lighten($base-label-color, 7.84) !default;
 
-  $chat-typingindicator-color: $chat-information-color !default;
   $chat-typingindicator-circle-bg-color: lighten($base-bg, 33.92) !default;
   $chat-typingindicator-circle-bg-color-center: lighten($base-bg, 43.92) !default;
 }
+
+/**
+* $name 80. Typing indicator color
```

Comment:
```suggestion
* $name 80. Typing indicator text color
```
I'm not so sure about this one, @EugeniyKiyashko can you please confirm that it's *text* color?

Reply:
Yes, it's color of the this text 
<img width="225" alt="image" src="https://github.com/user-attachments/assets/d266db92-da76-4ddc-ab0a-5aacfe3e8022">

Reply:
928565d1f0814077eb111893035b73aa0945b67b